ui/components/resolvers/ResolversContainer.js

```diff
@@ -24,24 +25,25 @@ class ResolversContainer extends Component {
         // TODO
     }
 
-    onResolverClicked(type, field, resolver) {
-        // TODO
-        console.log(type, field, resolver);
+    onResolverClicked(resolver) {
+        this.setState({ resolver });
     }
 
     render() {
+        const { resolver } = this.state;
+
         return (
             <React.Fragment>
                 <CommonToolbar
                     buttons={this.getToolbarButtons()}
                     onFilter={name => this.setFilter(name)}
                 />
                 <div className={styles.flexWrapper}>
-                    <div className={styles.left}>
-                        <ResolversList onClick={this.onResolverClicked} />
+                    <div className={styles.resolversListContainer}>
+                        <ResolversList onClick={res => this.onResolverClicked(res)} />
```

Comment:
This passes you three arguments and should look like: `(type, field, resolver) => this.onResolverClicked(type, field, resolver)`

Reply:
It's being taken care of https://github.com/aerogear/data-sync-ui/pull/80/files#diff-46ef7d14318a2e0fa30f96b90956812dR10.